mac

```diff
@@ -79,6 +79,22 @@ case "$SHELL" in
     ;;
 esac
 
+# checks architecture
+if [[ $(uname -m) == "arm64" ]]
+  then
+  echo "This Macbook has an M1 architecture."
```

Comment:
I don't feel strongly (don't have an M1) but this feels a bit noisy. Up to you whether you think otherwise.

Reply:
You are right, the cleaner the better.